src/lib/minions/data/killableMonsters/bosses/misc.ts

```diff
@@ -704,14 +706,28 @@ const killableBosses: KillableMonster[] = [
 			if (loot.has('Coagulated venom') && (ownedItems.has('Coagulated venom') || ownedItems.has('Rax'))) {
 				loot.set('Coagulated venom', 0);
 			}
+
+			loot.addItem(getOSItem('Noxious point').id, 4);
+			loot.addItem(getOSItem('Herbi').id, 3);
```

Comment:
?

Reply:
I was using that to debug, I removed it.